tests/Doctrine/Tests/DBAL/Schema/SqliteSchemaDiffTest.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+namespace Doctrine\Tests\DBAL\Schema;
+
+use Doctrine\DBAL\DBALException;
+use Doctrine\DBAL\Platforms\AbstractPlatform;
+use Doctrine\DBAL\Schema\ForeignKeyConstraint;
+use Doctrine\DBAL\Schema\SchemaDiff;
+use Doctrine\DBAL\Schema\Table;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+
+class SqliteSchemaDiffTest extends TestCase
+{
+    public function testSchemaDiffToSql() : void
+    {
+        $diff     = $this->createSchemaDiff();
+        $platform = $this->createPlatform(true);
+
+        $sql = $diff->toSql($platform);
+
+        $expected = ['create_table'];
+
+        self::assertEquals($expected, $sql);
+    }
+
+    public function testSchemaDiffToSaveSql() : void
+    {
+        $diff     = $this->createSchemaDiff();
+        $platform = $this->createPlatform(false);
+
+        $sql = $diff->toSaveSql($platform);
+
+        $expected = ['create_table'];
+
+        self::assertEquals($expected, $sql);
```

Comment:
This looks exactly as the test above. Instead of having the reader read both tests to notice that, please use one test with a data provider. The data provider should have keys describing what `true` and `false` mean.